src/View/Helper/GlideHelper.php

```diff
@@ -81,7 +81,11 @@ public function url($path, array $params = [])
         }
         $url = $this->urlBuilder()->getUrl($path, $params);
         if ($base && strpos($url, 'http') !== 0) {
-            $url = $this->request->getAttribute('webroot') . ltrim($url, '/');
+            if (method_exists($this->getView(), 'getRequest')) {
+                $url = $this->getView()->getRequest()->getAttribute('webroot') . ltrim($url, '/');
+            } else {
+                $url = $this->request->getAttribute('webroot') . ltrim($url, '/');
```

Comment:
I wouldn't have to do this string concatenation if the theme check in `UrlHelper::webroot()` could be turned off. Using it would cause 2 unnecessary `file_exists()` check for each image when using a theme :slightly_frowning_face:.